composer.json

```diff
@@ -0,0 +1,17 @@
+{
+    "name": "hlohaus/she-advanced-tinymce",
+    "keywords": [
+      "shopware",
+      "tinymce"
+    ],
+    "description": "Advanced TinyMCE settings",
+    "license": "MIT",
+    "type": "shopware-plugin",
+    "extra": {
+      "installer-name": "SheAdvancedTinyMce"
+    },
+    "require": {
+      "php": "^7.0",
```

Comment:
It's require php 7. shopware 5 is php 5.6 too

Reply:
set it to 5.6

Reply:
sorry, has to be `>=5.6` of cause

Reply:
yo have to update the look too

Reply:
done